src/BuildScriptGenerator/DotNetCore/ProjectFileProviders/ProbeAndFindProjectFileProvider.cs

```diff
@@ -83,6 +88,11 @@ public string GetRelativePathToProjectFile(RepositoryContext context)
                 projectFile = GetProject(azureFunctionsProjects);
             }
 
+            if (projectFile == null)
```

Comment:
Just realized we have a bug here. We need to ideally pass the combination (AddRange) of all these projects to see if there is more than 1 project that we found to build...or let's say we can look for a setting called 'ORYX_APP_TYPE' which can give us a hint on what type of app we are trying to build..if the app type is either functions or staticwebapps, then we only consider those apps

Reply:
I added some logic to deal with this..  please take a look